lapis2/src/main/kotlin/org/genspectrum/lapis/OpenApiDocs.kt

```diff
@@ -242,6 +254,35 @@ private fun aminoAcidMutations() =
                 ),
         )
 
+private fun nucleotideInsertions() =
+    Schema<List<NucleotideInsertion>>()
+        .type("array")
+        .items(
+            Schema<String>()
+                .type("string")
+                .example("ins_123:ATT")
+                .description(
+                    """
+                    |A nucleotide insertion in the format "ins_\<sequenceName\>?:\<position\>?:\<insertion\>?".  
```

Comment:
Isn't this slightly more correct?
```suggestion
                    |A nucleotide insertion in the format "ins_(\<sequenceName\>:)?\<position\>:\<insertion\>".  
```